public/js/hotkey_binder/model/forElements.js

```diff
@@ -0,0 +1,11 @@
+// const selectElement = (ele) => () => {
+// 	if (simulationArea.lastSelected && simulationArea.lastSelected.newElement)
+// 		simulationArea.lastSelected.delete();
+// 	const obj = new window[ele]();
+// 	simulationArea.lastSelected = obj;
+// 	smartDropXX += 70;
+// 	if (smartDropXX / globalScope.scale > width) {
+// 		smartDropXX = 50;
+// 		smartDropYY += 80;
+// 	}
+// };
```

Comment:
If this whole file is commented, can we remove it?

Reply:
Of course, I kept it incase it might it is decided to bring in elements drop shortcut, I shall remove this file.